web/templates/web/study-detail.html

```diff
@@ -43,10 +43,12 @@
             document.getElementById('too-young').style.display = 'none';
             var birthday = selected.value.split('|')[1];
             var age = calculateAgeInDays(birthday);
-            if (!isEligible(age)) {
-                if (isOldEnough(age)) {
```

Comment:
If you are not going to use the isEligible and isOldEnough methods since you need the `ineligible` count, I would remove those methods below.

Reply:
Done - thanks!